content/en/docs/languages/java/getting-started.md

```diff
@@ -260,10 +260,8 @@ For more:
 [logs]: /docs/concepts/signals/logs/
 [annotations]: /docs/zero-code/java/agent/annotations/
 [configure the java agent]: /docs/zero-code/java/agent/configuration/
-[console exporter]:
-  https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#logging-exporter
-[exporter]:
-  https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters
+[console exporter]: /docs/languages/java/configuration/
```

Comment:
I think this one should point at the exporters section as well, it's the only place in the doc that covers the console exporter
```suggestion
[console exporter]: /docs/languages/java/configuration/#properties-exporters
```